### DISCUSSION TITLE
how to view overset？

### DISCUSSION THREAD

**Discussion by Zcaic** [mdolab/MACH-Aero#112] [General]
I followed the tutorial， Successfully generated overlay mesh “ONERA_M6_L1.cgns” and “ONERA_M6_L1_IHC.cgns”. Then how can I view overset？ I used paraview， but there is only one variable “Mach”.

**Comment by hntter** [comment]
use tecplot. click 'plot' button, then choose 'blanking'->'value blanking', after that check 'include value blanking' and 'active'
